README.md

```diff
@@ -1,3 +1,5 @@
 # colaborativo-LED-continental
 
-Este es un ejemplo de como podemos trabajar de manera colaborativa con git.
\ No newline at end of file
+Este es un ejemplo de como podemos trabajar de manera colaborativa con git.
+
+Update 2 (Homework 3)
```

Comment:
What do you mean with Homework 3?

Reply:
Activity number 3, block two of the training